### DISCUSSION TITLE
compare files and export patch

### DISCUSSION THREAD

**Discussion by denimjeans** [digitalw0lf/hextor#44] [Ideas]
Hi Digitalw0lf, 

great job, like hextor really much. 
Is there a chance that you implement on the compare tab:
1. export patch (oldfile  new file patchfilename)
2. clickable list to see the differences and on click jump to the difference
3. option to copy the file offset address (right mouse button, copy address)

Patchfile output can look like this?:
0008302A:00->01
00083031:00->01
00083038:00->01
0008303F:00->01
00083046:00->01

regards
DJ

**Comment by digitalw0lf** [comment]
Hello,
Thanks for your feedback.
I'll see if I will have enough spare time to get back to Hextor development.
I already thought about "export patch" feature, and searched for some standardized patch format, but seems it only exists for text files (diff). All existing binary patch formats are complicated and not human readable.
How do you want to use these patches?

**Reply by denimjeans** [reply]
x64dbg :-) 
for malware analysis
for me text is fine. 
example: 
```
>rightfile.bin
<leftfile.bin
00083046:00->01
```
OFFSET    : FROM HEX -> TO HEX
